src/DeepLinkModule/javasource/deeplink/implementation/handler/DeeplinkHandler.java

```diff
@@ -0,0 +1,253 @@
+package deeplink.implementation.handler;

```

Comment:
How much of this is new/rewritten code, vs just moved? I haven't gone over every line yet; I'd like to get a feel for how thorough this review should be.

Reply:
Yes, I did extensively test it, but you could also say `I tested my own stuff`. 
Did drip feed the module to two teams who also tested their functionality with the new module. Both teams do have designated quality assurance members in the team.